docs/data/charts/quickstart/quickstart.md

```diff
@@ -72,51 +83,48 @@ Subcomponents include:
 
 - Axis components – to define the X and Y axes
 - Plot components – to create Bars, Lines, or any other Chart type
-- Auxillary components - to add Tooltips, Highlights, and more
-- Utilities - such as classes and types
+- Auxiliary components – to add Tooltips, Highlights, and more
+- Utilities – such as classes and types
 
 See the [Charts composition documentation](/x/react-charts/composition/) for complete details.
 
 The demo below shows how to use composition to create a custom Chart that combines a Bar and a Line Chart on a single plot:
 
 {{"demo": "Combining.js"}}
 
-## Chart layouts
+### Chart layouts
 
 The layout of a Chart is defined by two main spaces: the plot area, and the outer margins.
 
 The `width` and `height` props define the dimensions of the SVG which is the root of the chart.
-Within this SVG, the plot area (or drawing area) serves as the canvas for data visualization, where the lines, bars or other visual elements are rendered.
+Within this SVG, the plot area (or drawing area) serves as the canvas for data visualization, where the lines, bars, or other visual elements are rendered.
 The size of the plot area is determined by the `margin = {top, bottom, left, right}` object which defines its outer margins inside the SVG.
 The outer margin space is where information like axes, titles, and legends are displayed.
 
 See the [Styling documentation](/x/react-charts/styling/#placement) for complete details.
 
-## Server-side rendering
+### Server-side rendering
 
-The chart support server-side rendering under two conditions:
+Charts support server-side rendering under two conditions:
 
-1. The `width` and `height` props needs to be provided.
-2. The animation should be disabled with the `skipAnimation` prop.
+1. `width` and `height` props must be provided – it's not possible to compute the SVG dimensions on the server.
+2. Animations must be disabled with the `skipAnimation` prop – otherwise the animation will be in an empty state on first render.
```

Comment:
To avoid the "will" but not sure
```suggestion
2. Animations must be disabled with the `skipAnimation` prop – otherwise the first render show the initial frame of the animation.
```